lib/parsers.js

```diff
@@ -45,10 +45,12 @@ module.exports.parseError = function parseError(err, kwargs, cb) {
       kwargs.extra[name] = extraErrorProps;
     }
 
-    for (var n = frames.length - 1; n >= 0; n--) {
-      if (frames[n].in_app) {
-        kwargs.culprit = kwargs.culprit || utils.getCulprit(frames[n]);
-        break;
+    if (!kwargs.transaction && !kwargs.culprit) {
+      for (var n = frames.length - 1; n >= 0; n--) {
+        if (frames[n].in_app) {
+          kwargs.transaction = utils.getTransaction(frames[n]);
```

Comment:
Changed the logic here to only include a `transaction` if the user hasn't set a `culprit` manually. 
@mitsuhiko  should we add the transaction always - even if a manual culprit is given?